### ISSUE TITLE
Response when press back button from App password screen

### ISSUE THREAD

**Issue by Kuuuna98** [nextcloud/android#7343]
### Is your feature request related to a problem? 
After set app password, I can't exit the app by pressing the Back button on the password screen.
I thought there was an error in the app because there was no response even when I pressed the back button.

Android Naxtcloud app v. 3.13.1.
Android: 8
Device: Samsung Galaxy S7

### Describe the solution you'd like
Like the home screen of the app, it would be nice to leave the app by pressing the back button on the password screen. 
Or, when you press the Back button on the password screen, I would like to see a message telling you to press the Home button or the Menu button.

**Comment by tobiasKaminsky** [comment]
After setting app pin for first time (you have to type it in 2x), it should directly close the screen.

Can you show use a screenshot where it happens?

**Comment by Kuuuna98** [comment]
It's not a problem when setting up app pin.

reference video: https://drive.google.com/file/d/15jgc5_bht-npzcSkCTjyx46D9_tsruFY/view?usp=sharing

It meant that it would be nice to close the screen or see a message when I pressed the back button on the pin screen that I could see when I entered the app again after the app pin setup was complete. (reference video 0s-13s)
If I press the back button of the app while not locked, it will close the screen. (reference video 42s-46s)


Android Naxtcloud app v 3.15.0 Alpha1
Android: 5
Device: Samsung Galaxy S9

I tested it with an emulator to record the screen. The result is the same as Galaxy S7.
